# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xemacs-beta/fontconfig.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xemacs-21.5.28/src/font-mgr.c.vanilla	2009-02-23 15:02:04.000000000 +0100
+++ xemacs-21.5.28/src/font-mgr.c	2009-02-23 15:08:18.000000000 +0100
@@ -898,10 +898,8 @@
 		intern ("fc-config-get-blanks"));
 }
 
-/* The misspelling in the fontconfig function name accurately corresponds to
-   the version of fontconfig.h I had on 2007-04-13. -- sjt */
 DEFUN("fc-config-get-rescan-interval", Ffc_config_get_rescan_interval, 1, 1, 0, /*
- -- Function: int FcConfigGetRescanInverval (FcConfig *config)
+ -- Function: int FcConfigGetRescanItverval (FcConfig *config)
      Returns the interval between automatic checks of the configuration
      (in seconds) specified in 'config'.  The configuration is checked
      during a call to FcFontList when this interval has passed since
@@ -909,13 +907,11 @@
       (config))
 {
   CHECK_FCCONFIG (config);
-  return make_int (FcConfigGetRescanInverval (XFCCONFIG_PTR (config)));
+  return make_int (FcConfigGetRescanInterval (XFCCONFIG_PTR (config)));
 }
 
-/* The misspelling in the fontconfig function name accurately corresponds to
-   the version of fontconfig.h I had on 2007-04-13. -- sjt */
 DEFUN("fc-config-set-rescan-interval", Ffc_config_set_rescan_interval, 2, 2, 0, /*
- -- Function: FcBool FcConfigSetRescanInverval (FcConfig *config, int
+ -- Function: FcBool FcConfigSetRescanInterval (FcConfig *config, int
           rescanInterval)
      Sets the rescan interval; returns FcFalse if an error occurred.
      XEmacs: signal such error, or return nil on success. */
@@ -923,9 +919,9 @@
 {
   CHECK_FCCONFIG (config);
   CHECK_INT (rescan_interval);
-  if (FcConfigSetRescanInverval (XFCCONFIG_PTR (config),
+  if (FcConfigSetRescanInterval (XFCCONFIG_PTR (config),
 				 XINT (rescan_interval)) == FcFalse)
-    signal_error (Qio_error, "FcConfigSetRescanInverval barfed",
+    signal_error (Qio_error, "FcConfigSetRescanInterval barfed",
 		  intern ("fc-config-set-rescan-interval"));
   return Qnil;
 }
